### PR TITLE
Fix for "whiteSpace" typo: Parameter name and word is one command word spelled "whitespace".

### PR DESCRIPTION
**Pull request by ghost**
When browsing the vscode source, I noticed there is a misspelling of the word "whitespace", where the variable/property names were "whiteSpace". I noticed that this could have huge consequences, as the property "renderWhitespace" has no capital s, but in the commonEditorConfig, the property name has a capital s. As a result, I replaced "whiteSpace" variable names to "whitespace".

I did this by searching and replacing all instances of "whiteSpace" and "WhiteSpace" to "whitespace" and "Whitespace" respectively. There was one comment that was an incomplete sentence regarding one of the whitespace functions, so I fixed that as well.

Finally, I was able to reduce the code in PathCompletion regarding whitespace handling. The code has the same behavior, and passed the automated and smoke tests.

I hope this proves useful in the long run!

### CODE REVIEW

File: extensions/css-language-features/server/src/pathCompletion.ts [hunk 1]
```diff
@@ -144,27 +144,24 @@ const isDir = (p: string) => {
 };
 
 function pathToReplaceRange(valueBeforeCursor: string, fullValue: string, fullValueRange: Range) {
-	let replaceRange: Range;
```
Comment (mjbvz) [inline extensions/css-language-features/server/src/pathCompletion.ts]: Please revert the unrelated style changes. You can submit them as a separate PR if you feel they are worthwhile but they should not be part of this PR

Comment (relfarias) [reply extensions/css-language-features/server/src/pathCompletion.ts]: @mjbvz there is a previous pr opened to fix the issue on localisation key to match with the correct property. It might be related with the bug in how the selection boundaries are validated (potentially affects settings code).

Is it relevant to change other files in the project just to follow the lowercase  pattern used? (whitespace instead of whiteSpace)? these files are creating constants and being used properly (it doesn't affect the bug).

should be a different pr (refactor)? unless the bug is related to that

**Comment by pi1024e** [comment]
I undid the unnecessary style changes. Thank you.

File: src/vs/base/browser/ui/dialog/dialog.ts [hunk 2]
```diff
@@ -82,7 +82,7 @@ export class Dialog extends Disposable {
 		this.focusToReturn = document.activeElement as HTMLElement;
 
 		return new Promise<number>((resolve) => {
-			if (!this.element || !this.buttonsContainer || !this.iconElement || !this.toolbarContainer) {
```
Comment (mjbvz) [inline src/vs/base/browser/ui/dialog/dialog.ts]: Please revert these changes too. Again, this PR should only have spelling fixes and not any other refactorings

**Comment by pi1024e** [comment]
Are these changes appropriate, or are there any changes I need to fix?

**Comment by pi1024e** [comment]
Are these changes appropriate, or are there any changes I need to fix?

**Comment by mjbvz** [comment]
It looks like there are still refactoring changes in `rangeMap.ts` and `glob.ts`. Again, this change should only have the spelling fixes.

File: src/vs/base/browser/ui/list/listWidget.ts [hunk 3]
```diff
@@ -236,7 +236,7 @@ class KeyboardController<T> implements IDisposable {
 		private view: ListView<T>,
 		options: IListOptions<T>
 	) {
-		const multipleSelectionSupport = !(options.multipleSelectionSupport === false);
+		const multipleSelectionSupport = options.multipleSelectionSupport !== false;
```
Comment (pi1024e) [inline src/vs/base/browser/ui/list/listWidget.ts]: I will do the other programming changes requested, but this the one unrelated line is not only poor style, but the only one of its kind (every other statement in this entire program like this uses !==). I understand why the other refactorings should be in a different PR and undone but this sticks out like a sore thumb.

Only time I will ever say this.

**Comment by pi1024e** [comment]
Made changes!

**Comment by mjbvz** [comment]
Thanks!

**Comment by Tyriar** [comment]
@pi1024e the way you're working is spamming links in some of our issues, for example in https://github.com/microsoft/vscode/issues/78168:

image

It looks like you're co-authoring commits when you're syncing or something, please avoid this as it causes a lot of noise.
